debezium-server/debezium-server-core/src/test/java/io/debezium/server/DebeziumServerIT.java

```diff
@@ -26,32 +26,22 @@
  * @author Jiri Pechanec
  */
 @QuarkusTest
+@QuarkusTestResource(TestDatabase.class)
 public class DebeziumServerIT {
 
     private static final int MESSAGE_COUNT = 4;
-    protected static TestDatabase db = null;
+    @Inject
+    DebeziumServer server;
```

Comment:
Where is this used?

Reply:
not 100% sure but, i believe this needed to start debezium-server and consume events, test that events are generated.
used in line 56
`        final TestConsumer testConsumer = (TestConsumer) server.getConsumer();`